### PR TITLE
#152796588 configure eslint for linting

### PR DESCRIPTION
**Pull request by HamdalahA**
What does this PR do?
Configure eslint

Description of Task to be completed?
- [x] Install eslint globally and configure eslint for folder by creating an eslint file in your project root folder

How should this be manually tested?
- [x] Run `npm run eslint` in console
- [x] check files to see if eslint is running
- [x] check hound report on pull request.

What are the relevant pivotal tracker stories?
[152796588](https://www.pivotaltracker.com/story/show/152796588)

Screenshot showing eslint in action
screen shot 2017-11-21 at 12 46 41

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,6 +41,7 @@
     "passport-google-oauth": "~0.1.5",
     "passport-local": "~0.1.6",
     "passport-twitter": "~1.0.2",
+    "react-router-dom": "^4.2.2",
```
Comment (monsieurkayode) [inline package.json]: Please what is this library doing here?

**Review by proxiex — APPROVED** [review]

File: .eslintrc.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
```
Comment (monsieurkayode) [inline .eslintrc.yml]: Can you rename this to file to .eslintrc instead
Reason being that the gulp task runner for linting will be looking for a .eslintrc file in the root directory to apply rules

File: .eslintrc.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  "root": true,
+  "extends": "airbnb-base",
+  "env": {
+    "node": true,
+    "es6": true,
+    "mocha": true
+  },
```
Comment (monsieurkayode) [inline .eslintrc.yml]: Can you also  add parser options for ecmaVersion 6 in this config file to fix Hound CI flagging arrow functions

**Review by monsieurkayode — APPROVED** [review]
We are good here

**Review by andela-moe — COMMENTED** [review]
Shouldn't this file `.vscode/settings.json` be ignored?

**Pull request closed without merging**